src/services/utils/jwt.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs'
```

Comment:
For consistency, I’d suggest moving JwtService from the utils directory to services.

Reply:
no problem